### PR TITLE
Fix loading PSD layer with non-ascii name

### PR DESCRIPTION
**Pull request by shun-iwasawa**
This PR is for the issue #309 
Please note that I could modify only for Windows. For OS X the sources are left unchanged.

### CODE REVIEW

**Comment by shun-iwasawa** [comment]
Jenkins

File: toonz/sources/image/psd/tiio_psd.cpp [hunk 1]
```diff
@@ -40,7 +40,11 @@ TLevelReaderPsd::TLevelReaderPsd(const TFilePath &path)
 #ifdef REF_LAYER_BY_NAME
 		if (layerStr != "frames") {
 			TPSDParser psdparser(m_path);
+#ifdef _WIN32
+			m_layerId = psdparser.getLevelIdByName(layerStr.toLocal8Bit().constData());
```
Comment (kogaki) [inline toonz/sources/image/psd/tiio_psd.cpp]: Why do you use `constData` instead of `toStdString` ?

**Comment by kogaki** [comment]
@shun-iwasawa  This problem is not related to `local system` encoding but `psd file` encoding. I created alternative PR #355 . Plz review it. Thanks.

**Comment by shun-iwasawa** [comment]
@kogaki did the better resolution in PR #355 
Thanks!

**Pull request closed without merging**